internal/app/http/handler.go

```diff
@@ -365,15 +361,15 @@ func (h *TmcHandler) DeleteTMNameAttachment(w http.ResponseWriter, r *http.Reque
 
 func (h *TmcHandler) PutThingModelAttachmentByName(w http.ResponseWriter, r *http.Request, tmID string, attachmentFileName string, params server.PutThingModelAttachmentByNameParams) {
```

Comment:
To make it clear and align the naming with putting attachments to a TM-name (PutTMNameAttachment), I suggest to rename it to PutTMIDAttachment.

Reply:
done